src/agent/tests/CMakeLists.txt

```diff
@@ -45,3 +45,11 @@ if(UNIX AND NOT APPLE)
     target_link_libraries(unix_daemon_test PRIVATE Agent GTest::gtest)
     add_test(NAME UnixDaemonTest COMMAND unix_daemon_test)
 endif()
+
+if(WIN32)
+    add_executable(windows_service_test windows_service_test.cpp)
+    configure_target(windows_service_test)
+    target_include_directories(windows_service_test PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../include ${CMAKE_CURRENT_SOURCE_DIR}/../src)
```

Comment:
Since you are including the Agent target, is not necessary to add the include directory here (see https://github.com/wazuh/wazuh-agent/pull/276 there's a commit fixing the usage of the `target_include_directories` cmake command).

Reply:
Fixed!